src/routes/custom-routes-food.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const express = require('express');
+const foodSchema = require('../models/food-schema.js')
+const DataCollection = require('../models/data-collection-class.js');
+const food = new DataCollection(foodSchema);
+
+const foodRouter = express.Router();
+
+foodRouter.get('/:id', getOneFood)
+foodRouter.post(createFood)
+foodRouter.put('/:id', updateFood)
+foodRouter.delete('/:id', deleteFood)
```

Comment:
Add 'food' to your routes like below. That way the other model won't get confused
```
foodRouter.get('/food',getFood);
foodRouter.get('/food/:id', getOneFood);
foodRouter.post('/food',createFood);
foodRouter.put('food/:id', updateFood);
foodRouter.delete('food/:id', deleteFood);
```